upcoming_changes/2972.maintenance.rst

```diff
@@ -0,0 +1 @@
+RosettaSciIO was split out of the `HyperSpy repository <https://github.com/hyperspy/hyperspy>`_ on July 23, 2022. The IO-plugins and related functions so far developed in HyperSpy were moved to the `RosettaSciIO repository <https://github.com/hyperspy/rosettasciio>`_.
```

Comment:
Would it be better to classify this note as API change?

Reply:
I was indeed hesitating, which category to choose:
- `maintenance` is defined as concerning packaging, which I would say this one does.
- And the actual usage of the HyperSpy-API for the user does not change (they just have to install an additional package), which is why I hesitated to categorize it as `api`.

That said about my line of thought, I am open to categorize it differently.

Reply:
I suspect that API which is not in `hyperspy.api`, for example, `from hyperspy.io_plugins.msa import file_reader` is being used and in that sense it breaks the API.
Even if this changes will be transparent to the large majority of users, I think that this is useful to flag it at API changes to be on the safe side!

In any case, the 2.0 release will most likely not be as automated as with other and the changelog and the wording of the changelog will be tweaked before the release!

Reply:
Fair argument, I changed it from `maintenance` to `api`